core/perf_test/CMakeLists.txt

```diff
@@ -114,3 +114,91 @@ IF(NOT Kokkos_ENABLE_OPENMPTARGET)
     CATEGORIES PERFORMANCE
   )
 ENDIF()
+
+
+IF(NOT Kokkos_ENABLE_BENCHMARKS OR KOKKOS_HAS_TRILINOS)
```

Comment:
Disallow enabling benchmarks when using `Trilinos` instead of silently doing nothing.

Reply:
Done (7f10e39799487c7ed30157a27a94764914960b8a).